src/nationGen/naming/NationDescriber.java

```diff
@@ -52,14 +52,6 @@ private void describeUnits(DescriptionReplacer dr, List<Unit> units, Filter desc
 				.collect(Collectors.joining(" "))).append(" ");
 				
 			
-			for(Filter f : u.appliedFilters)
-			{
-				if(f != null)
-				{
-					desc.append(f.tags.getString("description").map(s -> s + " ").orElse(""));
-				}
-			}
-			
```

Comment:
Can you describe what this change achieves?  I'm not really sure what the purpose of this method is

Reply:
This is the method that generates the description for individual troops. This loop in particular appends a brief description for each of the current unit's filters (e.g. "Poison resistant", "Strong", etc.) This same loop shows up identically earlier in the method, with nothing between the two loops that would modify what they do, serving no apparent purpose other than duplicating the filter descriptions. This results in unit descriptions such as "The infantry of foo are armed with various weapons and light armor. Poison resistant Strong Poison resistant Strong", which gets even worse when you get into filters that have longer descriptions. This change removes the redundant loop to make troop descriptions a little more readable. (This only affects troop descriptions. Commander descriptions are set in a separate method which does not feature the redundant loop.)

Reply:
Gotcha, I see it